scripts/build/TestFx.props

```diff
@@ -8,6 +8,12 @@
     <IsTest Condition=" '$(IsTest)' == '' ">false</IsTest>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <BranchName Condition=" '$(BranchName)' == '' ">BRANCHNAME</BranchName>
+    <CommitHash Condition=" '$(CommitHash)' == '' ">COMMITHASH</CommitHash>
+    <CommitId>$(CommitHash.SubString(0, 8))</CommitId>
```

Comment:
I understand that trimming the commit hash makes it easier to look at, but should we use the full commit hash instead ? I know it shouldn't come to it but I'm wondering what happens in case of a collision.

Reply:
It won't happen. (Too low of a chance to happen.) But in that case we also store full commit hash in binary too.